### DISCUSSION TITLE
Mounting zFS filesystems with automatic growing

### DISCUSSION THREAD

**Discussion by plavjanik** [zowe/zowe-cli#873] [Q&A] [answered]
The z/OS `mount` command allows passing additional options via `-o`, for example: `-o aggrgrow` that allows the zFS to grow automatically. Is it possible to pass this option in Zowe CLI when the mount is done using `zowe zos-files mount file-system`?

**Comment by MikeBauerCA** [comment]
This does not appear to be supported by the z/OSMF REST API that is currently being leveraged: https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.izua700/IZUHPINFO_API_MountUnixFile.htm

**Comment by zFernand0** [comment]
Is this a good candidate to convert to a Discussion? 😋

**Reply by MikeBauerCA** [reply]
Our first migrated issue! Good idea @zFernand0

**Reply by plavjanik** [reply]
Thanks for looking at it. What is your plan to address the issue? The Zowe mount command is not useful for me if it does not support automatic growing. My workaround is to use ssh where I can issue the proper mount command. Would not it be possible for Zowe CLI to use ssh or TSO under the covers instead of the limited z/OSMF REST API?

**Comment by plavjanik** [comment, marked as answer]
Workaround: 

Instead of using Zowe CLI mount command that does not support specifying additional parameter using SSH via Zowe CLI:

```sh
zowe zos-uss issue ssh "/usr/sbin/mount -o aggrgrow -f <fileSystemName> <mountPoint>"
```

**Comment by MikeBauerCA** [comment]
While it is a good workaround, SSH requires additional setup, e.g. an ssh profile, while other commands in the `zos-files` group do not require this information. Perhaps we should open an enhancement request with z/OSMF. What do you think of this approach?

**Reply by plavjanik** [reply]
I see. It would be ideal if z/OSMF REST API support that. Opening RFE for z/OSMF makes the best sense.
